examples/jetreco.jl

```diff
@@ -140,17 +140,19 @@ function main()
     logger = ConsoleLogger(stdout, Logging.Info)
     global_logger(logger)
     # Try to read events into the correct type!
-    if JetReconstruction.is_ee(args[:algorithm])
-        jet_type = EEjet
+    # If we don't have an algorithm we default to PseudoJet
+    if !isnothing(args[:algorithm])
+        JetReconstruction.is_ee(args[:algorithm])
+        jet_type = EEjet{Float64}
```

Comment:
just a question: why is it explicitly `Float64` here, shouldn't this be smth like `eval(args[:type])` for generality?

Reply:
The reason is that I wanted `jetreco.jl` to be the simplest and most explicit example possible. `instrumented-jetreco.jl` is the one with all the bells and whistles.

It might be useful to add a note though, that any subtype of `Real` is allowed.